views/index.erb

```diff
@@ -11,10 +11,10 @@
         </h2>
         <ul class="pulls">
           <% application.fetch(:pull_requests).each do |pull_request| %>
```

Comment:
Not sure if this is awful but would do the trick, not requiring changes to the backend objects

```
p "Opened 4 days ago"[/\d+/].to_i > 5 ? 'red' : ''
p "Opened 8 days ago"[/\d+/].to_i > 5 ? 'red' : ''
p "Opened Yesterday"[/\d+/].to_i > 5 ? 'red' : ''
p "Opened today"[/\d+/].to_i > 5 ? 'red' : ''
# >> ""
# >> "red"
# >> ""
# >> ""
```